src/engine/game/gui/ingame/character.h

```diff
@@ -18,23 +18,105 @@
+        std::shared_ptr<gui::Slider> sldAlign {nullptr};
+        std::shared_ptr<gui::Label> lblStr {nullptr};
+        std::shared_ptr<gui::Label> lblFortitudeStat {nullptr};
+        std::shared_ptr<gui::Label> lblReflexStat {nullptr};
+        std::shared_ptr<gui::Label> lblWillStat {nullptr};
+        std::shared_ptr<gui::Label> lblDefenseStat {nullptr};
+        std::shared_ptr<gui::Label> lblForceStat {nullptr};
+        std::shared_ptr<gui::Label> lblVitalityStat {nullptr};
+        std::shared_ptr<gui::Label> lblDex {nullptr};
+        std::shared_ptr<gui::Label> lblCon {nullptr};
+        std::shared_ptr<gui::Label> lblInt {nullptr};
+        std::shared_ptr<gui::Label> lblCha {nullptr};
+        std::shared_ptr<gui::Label> lblWis {nullptr};
+        std::shared_ptr<gui::Label> lblStrMod {nullptr};
+        std::shared_ptr<gui::Label> lblDexMod {nullptr};
+        std::shared_ptr<gui::Label> lblConMod {nullptr};
+        std::shared_ptr<gui::Label> lblIntMod {nullptr};
+        std::shared_ptr<gui::Label> lblWisMod {nullptr};
+        std::shared_ptr<gui::Label> lblChaMod {nullptr};
+        std::shared_ptr<gui::Label> lblExperienceStat {nullptr};
+        std::shared_ptr<gui::Label> lblNeededXp {nullptr};
+        std::shared_ptr<gui::Label> lblStrength {nullptr};
+        std::shared_ptr<gui::Label> lblDexterity {nullptr};
+        std::shared_ptr<gui::Label> lblConstitution {nullptr};
+        std::shared_ptr<gui::Label> lblIntelligence {nullptr};
+        std::shared_ptr<gui::Label> lblCharisma {nullptr};
+        std::shared_ptr<gui::Label> lblReflex {nullptr};
+        std::shared_ptr<gui::Label> lblWill {nullptr};
+        std::shared_ptr<gui::Label> lblExperience {nullptr};
+        std::shared_ptr<gui::Label> lblNextLevel {nullptr};
+        std::shared_ptr<gui::Label> lblForce {nullptr};
+        std::shared_ptr<gui::Label> lblVitality {nullptr};
+        std::shared_ptr<gui::Label> lblDefense {nullptr};
+        std::shared_ptr<gui::Label> lblFortitude {nullptr};
+        std::shared_ptr<gui::Label> lblBevel {nullptr};
+        std::shared_ptr<gui::Label> lblWisdom {nullptr};
+        std::shared_ptr<gui::Label> lblBevel2 {nullptr};
+        std::shared_ptr<gui::Label> lblLight {nullptr};
+        std::shared_ptr<gui::Label> lblDark {nullptr};
+        std::shared_ptr<gui::Button> btnExit {nullptr};
+        std::shared_ptr<gui::Button> btnAuto {nullptr};
+        std::shared_ptr<gui::Button> btnLevelup {nullptr};
+        std::shared_ptr<gui::Button> btnChange1 {nullptr};
+        std::shared_ptr<gui::Button> btnChange2 {nullptr};
+
+        // KOTOR only
+        std::shared_ptr<gui::Label> lblAdorn {nullptr};
+        std::shared_ptr<gui::Button> btnScripts {nullptr};
+        std::shared_ptr<gui::Label> lblClass {nullptr};
+        std::shared_ptr<gui::Label> lblClass1 {nullptr};
+        std::shared_ptr<gui::Label> lblClass2 {nullptr};
+        std::shared_ptr<gui::Label> lblGood[10];
```

Comment:
Since 10 and 6 below are being referenced in a loop, they should be extracted to constants:

```cpp
constexpr int kNumControlsGood = 10;
constexpr int kNumControlsBar = 6;
```